dto.c

```diff
@@ -1208,15 +1260,13 @@ static __always_inline  struct dto_wq *get_wq(void* buf)
 		int status[1] = {-1};
 
 		// get numa node of memory pointed by buf
```

Comment:
The comment doesn't look correct for cpu centric numa awareness. May be change is to "// get the numa node for the target DSA device"